src/lib/sensor_calibration/Magnetometer.cpp

```diff
@@ -110,7 +110,7 @@ void Magnetometer::SensorCorrectionsUpdate(bool force)
 
 bool Magnetometer::set_offset(const Vector3f &offset)
 {
-	if (Vector3f(_offset - offset).longerThan(0.01f)) {
+	if (Vector3f(_offset - offset).longerThan(0.001f)) {
```

Comment:
This was done to minimize writing out the parameters after every flight for negligible changes.

Reply:
Ok, I can revert this if you're fine with changing `magb_vref` (as modified in this PR). Otherwise the change is so small that even an offset of 0.1gs is never getting corrected (the change in offset is ~0.003gs in sim)

Reply:
Let's set it to 0.005 so it gets applied if the change in offset is > 1% of the usual mag field